src/components/Icon/Chevron.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from "react";
+import { SVGProps } from "react";
+
+export interface ChevronIconProps extends SVGProps<SVGSVGElement> {
+  direction: ChevronIconDirection
```

Comment:
nit: I like this, but as we've discussed, unsure if we could just have a global `Direction` enum, how likely is that the `Chevron` and others will have different directions?

no action needed, just food for thought